hudi-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestUtils.java

```diff
@@ -217,6 +239,51 @@ public static SparkConf getSparkConfForTest(String appName) {
     }
   }
 
+  public static Dataset<Row> readHFile(JavaSparkContext jsc, SQLContext sqlContext, String[] paths) {
+    // TODO: this should be ported to use HoodieStorageReader
+    List<byte[]> valuesAsList = new LinkedList<>();
+
+    FileSystem fs = FSUtils.getFs(paths[0], jsc.hadoopConfiguration());
+    CacheConfig cacheConfig = new CacheConfig(fs.getConf());
+    Schema schema = null;
+    for (String path : paths) {
```

Comment:
We might eventually need such support when supporting spark datasource support on Hoodie HFile table. It would be better to move this logic to a Utils class to reuse.